conda/core/path_actions.py

```diff
@@ -404,7 +404,9 @@ def verify(self):
 
         try:
             log.trace("rewriting prefixes in %s", self.target_full_path)
-            update_prefix(self.intermediate_path, self.target_prefix, self.prefix_placeholder,
+            update_prefix(self.intermediate_path,
+                          context.target_prefix_override or self.target_prefix,
+                          self.prefix_placeholder,
```

Comment:
Maybe we could use named args (no necessarily kwargs) so that this function (and other functions throughout the codebase are  easier to read/understand on first read?

```python
update_prefix(
    prefix=self.intermediate_path,
    new_prefix=context.target_prefix_override or self.target_prefix, 
    placeholder=self.prefix_placeholder,
    mode=self.file_mode,
)
```

Just wondering what you think about this in general @kalefranz. Not a request for change :-p, but a request for comment ;-)

Reply:
I like that idea. I’ll try to start doing that.